modules/standard/Map.chpl

```diff
@@ -284,8 +310,16 @@ module Map {
                this map.
     */
     iter items() const ref {
-      for key in myKeys {
```

Comment:
I think we need to split this into two overloads as was done below for the `values` iterator, due to the cast returning a reference to a temporary.